bin/clean-tsv-metadata

```diff
@@ -14,6 +14,11 @@ def clean_metadata_file(file_name, n_cols, header):
                            header=0 if header else None, #needs to happen if a header already exists and you are overriding with "names"
                            names=col_names, # overriding column names; this adds missing tabs when there are too few
                            usecols=col_names) # using first n only; this removes extra tabs
+    if sort_col is not None:
+        if not header:
+            sort_col = int(sort_col) + 1
+        col_names.remove(sort_col)
+        col_names = [sort_col] + col_names
```

Comment:
You may want to consider how to handle ValueErrors that `col_names.remove()` throws in the event the given `sort_col` does not exist in the given TSV.

Reply:
Additionally, I'm not sure that this section of the code is doing what we want. e.g. 

```
head -n 5 source-data/gisaid_annotations.tsv > source-data/test_annotations.tsv 
tail -n 5 source-data/gisaid_annotations.tsv >> source-data/test_annotations.tsv 
cp source-data/test_annotations.tsv source-data/copy_test_annotations.tsv
./bin/clean-tsv-metadata --metadata source-data/copy_test_annotations.tsv --n-cols 4 --sort-col 3
```

results in the following error:
```
Traceback (most recent call last):
  File "./bin/clean-tsv-metadata", line 55, in <module>
    clean_metadata_file(args.metadata, args.n_cols, args.header, args.sort_col)
  File "./bin/clean-tsv-metadata", line 20, in clean_metadata_file
    col_names.remove(sort_col)
ValueError: list.remove(x): x not in list
```

Would you expect sorting by column `3` to work?

Reply:
Should be fixed by 74c4dcb!